test/fixtures/es-class-assign-property/expected-remove-context-wrap-es6.js

```diff
@@ -0,0 +1,23 @@
+class Foo1 extends React.Component {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+process.env.NODE_ENV !== "production" ? Foo1.childContextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+class Foo2 extends React.PureComponent {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
```

Comment:
Sorry, but I don't get what you are referring to :confused: , I think this is the correct behavior. This remove the `PropTypes` property, but only wrap `ContextTypes` and `childContextTypes`.